src/test/unit/VoteWeigherBaseUnit.t.sol

```diff
@@ -0,0 +1,664 @@
+        uint256[] memory indicesToRemove
+    ) public {
+        cheats.assume(notServiceManagerOwner != serviceManagerOwner);
+        strategiesAndWeightingMultipliers = _convertToValidStrategiesAndWeightingMultipliers(strategiesAndWeightingMultipliers);
+
+        // create a valid quorum
+        uint8 quorumNumber = voteWeigher.quorumCount();
+        cheats.prank(serviceManagerOwner);
+        voteWeigher.createQuorum(strategiesAndWeightingMultipliers);
+
+        // remove certain strategies
+        cheats.prank(notServiceManagerOwner);
+        cheats.expectRevert("VoteWeigherBase.onlyServiceManagerOwner: caller is not the owner of the serviceManager");
+        voteWeigher.removeStrategiesConsideredAndMultipliers(quorumNumber, indicesToRemove);
+    }
+
+    function testRemoveStrategiesConsideredAndMultipliers_ForNonExistantQuorum_Reverts(
+        IVoteWeigher.StrategyAndWeightingMultiplier[] memory strategiesAndWeightingMultipliers,
+        uint256[] memory indicesToRemove
+    ) public {
+        strategiesAndWeightingMultipliers = _convertToValidStrategiesAndWeightingMultipliers(strategiesAndWeightingMultipliers);
+
+        // create a valid quorum
+        uint8 quorumNumber = voteWeigher.quorumCount();
+        cheats.startPrank(serviceManagerOwner);
+        voteWeigher.createQuorum(strategiesAndWeightingMultipliers);
+
+        // remove strategies from a non-existant quorum
+        cheats.expectRevert("VoteWeigherBase.removeStrategiesConsideredAndMultipliers: quorumNumber is greater than or equal to quorumCount");
+        voteWeigher.removeStrategiesConsideredAndMultipliers(quorumNumber + 1, indicesToRemove);
+    }
+
+    function testRemoveStrategiesConsideredAndMultipliers_EmptyIndicesToRemove_HasNoEffect(
+        IVoteWeigher.StrategyAndWeightingMultiplier[] memory strategiesAndWeightingMultipliers
+    ) public {
+        strategiesAndWeightingMultipliers = _convertToValidStrategiesAndWeightingMultipliers(strategiesAndWeightingMultipliers);
+
+        // create a valid quorum
+        uint8 quorumNumber = voteWeigher.quorumCount();
+        cheats.startPrank(serviceManagerOwner);
+        voteWeigher.createQuorum(strategiesAndWeightingMultipliers);
+
+        // remove no strategies
+        voteWeigher.removeStrategiesConsideredAndMultipliers(quorumNumber, new uint256[](0));
+        // make sure the quorum strategies and weights haven't changed
+        for (uint i = 0; i < strategiesAndWeightingMultipliers.length; i++) {
+            IVoteWeigher.StrategyAndWeightingMultiplier memory strategyAndWeightingMultiplier = voteWeigher.strategyAndWeightingMultiplierForQuorumByIndex(quorumNumber, i);
+            assertEq(address(strategyAndWeightingMultiplier.strategy), address(strategiesAndWeightingMultipliers[i].strategy));
+            assertEq(strategyAndWeightingMultiplier.multiplier, strategiesAndWeightingMultipliers[i].multiplier);
+        }
+    }
+
```

Comment:
should we also have a test for trying to remove invalid indices? I'm presuming this would just break with an out-of-bounds access error?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/53/commits/43b1830872bd61cc16f74b9db576ca4bd0eec68a